records_mover/records/schema/field/sqlalchemy.py

```diff
@@ -167,30 +167,23 @@ def field_to_sqlalchemy_type(field: 'RecordsSchemaField',
     field_to_func_map = {
         'integer': get_integer_type,
         'decimal': get_decimal_type,
-        'boolean': cast(Callable[['RecordsSchemaField', 'DBDriver'], sqlalchemy.types.TypeEngine],
-                        lambda field, driver: sqlalchemy.sql.sqltypes.BOOLEAN()),
+        'boolean': lambda field, driver: sqlalchemy.sql.sqltypes.BOOLEAN(),
         'string': get_string_type,
-        'date': cast(Callable[['RecordsSchemaField', 'DBDriver'], sqlalchemy.types.TypeEngine],
-                     lambda field, driver: sqlalchemy.sql.sqltypes.DATE()),
-        'datetime': cast(Callable[['RecordsSchemaField', 'DBDriver'], sqlalchemy.types.TypeEngine],
-                         lambda field, driver: driver.type_for_date_plus_time(has_tz=False)),
-        'datetimetz': cast(
-                      Callable[['RecordsSchemaField', 'DBDriver'], sqlalchemy.types.TypeEngine],
-                      lambda field, driver: driver.type_for_date_plus_time(has_tz=True)),
-        'time': cast(Callable[['RecordsSchemaField', 'DBDriver'], sqlalchemy.types.TypeEngine],
-                     lambda field, driver: sqlalchemy.sql.sqltypes.TIME(timezone=False)
-                     if driver.supports_time_type()
-                     else sqlalchemy.sql.sqltypes.VARCHAR(8)),
-        'timetz': cast(Callable[['RecordsSchemaField', 'DBDriver'], sqlalchemy.types.TypeEngine],
-                       lambda field, driver: sqlalchemy.sql.sqltypes.TIME(timezone=True)
-                       if driver.supports_time_type()
-                       else sqlalchemy.sql.sqltypes.VARCHAR(8))
+        'date': lambda field, driver: sqlalchemy.sql.sqltypes.DATE(),
+        'datetime': lambda field, driver: driver.type_for_date_plus_time(has_tz=False),
+        'datetimetz': lambda field, driver: driver.type_for_date_plus_time(has_tz=True),
+        'time': lambda field, driver: sqlalchemy.sql.sqltypes.TIME(timezone=False)
+        if driver.supports_time_type() else sqlalchemy.sql.sqltypes.VARCHAR(8),
```

Comment:
these multi-line inline if statements are kinda weirdin me out, sure it doesn't need a backslash or somethnig?

Reply:
I thought so too but flake8 says this is where they go, I got a hanging indent error for trying to make them look reasonable.

Reply:
I can think about just putting everything in parens